p2p/sentry/simulator/sentry_simulator.go

```diff
@@ -0,0 +1,454 @@
+package simulator
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/ledgerwatch/erigon-lib/chain/snapcfg"
+	"github.com/ledgerwatch/erigon-lib/common"
+	"github.com/ledgerwatch/erigon-lib/downloader/snaptype"
+	"github.com/ledgerwatch/erigon-lib/gointerfaces"
+	sentry_if "github.com/ledgerwatch/erigon-lib/gointerfaces/sentry"
```

Comment:
fyi we were chatting with @battlmonstr about trying to stick to some common code style, like google's code style guide
 
one example was to stick to unbroken lowercase imports e.g.  https://google.github.io/styleguide/go/decisions#package-names

we are only starting the conversation now though (no action needed), will probably be best to introduce some linter changes later on once we have some alignment with everyone

Reply:
This is not a package name its a disambiguation.

I think you get better code readability if you use this model for name disambiguation because:

* its clear its not a package
* it gives some clue as to where the package came from

I think its better than the package1, package2 etc.

Reply:
I agree, it is better than package1, package2 to disambiguate, however it could be `sentryif` or `sentryinterfaces` - ie avoiding snake case